cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -51,7 +51,7 @@ var addonsInstallCmd = &cobra.Command{
 func init() {
 	addonsInstallCmd.Flags().BoolVar(&force, forceFlag, false, "Forces the installation of the add-on even if the add-on was previously installed.")
 	addonsInstallCmd.Flags().BoolVar(&enable, enableFlag, false, "If true, installs and enables the specified add-on with the default priority.")
-	addonsInstallCmd.Flags().BoolVar(&defaults, defaultsFlag, false, "If true, installs all Minishift default add-ons.")
+	addonsInstallCmd.Flags().BoolVar(&defaults, defaultsFlag, false, fmt.Sprintf("If true, installs all default add-ons.(default: %s)", strings.Join(util.DefaultAssets, ", ")))
```

Comment:
`fmt.Sprintf("If true, installs all default add-ons.(%s)", strings.Join(util.DefaultAssets, ", ")))`

I don't think `default` is required in brackets since we already mentioned ahead of it.

Reply:
Right! Changing..